tests/db/conftest.py

```diff
@@ -60,11 +59,12 @@ def cleanup_tables(request, node_config):
     def fin():
         conn = connect()
         try:
-            conn.run(r.db(db_name).table('bigchain').delete())
-            conn.run(r.db(db_name).table('backlog').delete())
-            conn.run(r.db(db_name).table('votes').delete())
-        except r.ReqlOpFailedError as e:
-            if e.message != 'Database `{}` does not exist.'.format(db_name):
+            schema.drop_database(conn, db_name)
+            schema.create_database(conn, db_name)
+            schema.create_tables(conn, db_name)
+            schema.create_indexes(conn, db_name)
```

Comment:
Not sure if we need to recreate the db, tables, and indexes again here; wouldn't `setup_database` handle this with `init_database()`?